pre_build.py

```diff
@@ -14,9 +14,12 @@ def build() -> None:
         )
         exit(1)
     print("Building front end...")
-    subprocess.run(["yarn", "webui:install"])
-    subprocess.run(["yarn", "webui:build"])
-
+    if os.name == "nt":
+        subprocess.run(" ".join(["yarn", "webui:install"]), shell=True)
```

Comment:
Not sure if it relates to the issues you're having, but you probably wanna do subprocess.check_output instead of .run here, otherwise you're silently ignoring failures.